setup.cfg

```diff
@@ -1,3 +1,158 @@
-[aliases]
-test=pytest
+[metadata]
+name = pypeit
+description = PypeIt Spectroscopic Reduction
+long_description = file: README.md
+author = PypeIt Collaboration
+author_email = pypeit@ucolick.org
+license = BSD-3
+url = https://github.com/pypeit/PypeIt
+edit_on_github = False
+github_project = pypeit/PypeIt
+keywords = pypeit, PypeIt, astronomy, Keck, UCO, Lick, spectroscopy, data reduction
+classifiers =
+    Development Status :: 4 - Beta
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Natural Language :: English
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3.7
+    Topic :: Documentation :: Sphinx
+    Topic :: Scientific/Engineering :: Astronomy
+    Topic :: Software Development :: Libraries :: Python Modules
+    Topic :: Software Development :: User Interfaces
 
+[options]
+zip_safe = False
+use_2to3=False
+packages = find:
+python_requires = >=3.7
+setup_requires = setuptools_scm
+include_package_data = True
+install_requires =
+    numpy>=1.18
+    astropy>=4.0
+    extension-helpers>=0.1
+    packaging>=0.19
+    scipy>=1.4
+    matplotlib>=3.1
+    PyYAML>=5.1
+    configobj>=5.0.6
+    scikit-learn>=0.2
+    IPython>=7.2.0
+    numba>=0.39.0
+    ginga>=3.0
+    linetools
+    qtpy>=1.9
+scripts =
+    bin/pypeit_c_enabled
+    bin/pypeit_chk_plugins
+    bin/pypeit_clean
+    bin/pypeit_version
+    bin/copy_spec1d.py
```

Comment:
I see you just moved this file, but can we rename it?  Something like `pypeit_cp_spec1d`, including the removal of the `.py` extension.  I prefer executable scripts to not have the extension.

Reply:
agreed. renamed the script and updated `setup.cfg`.